deployment/integration.yaml

```diff
@@ -8,5 +8,6 @@ portal:
   config:
     issuer: https://id.test.vshn.net/auth/realms/VSHN-main-dev-realm
     clientId: appuio-control-api
+    server: https://api-integration.apps.cloudscale-lpg-2.appuio.cloud/
```

Comment:
@tobru the `issuer` (used for `--oidc-issuer-url`) and `clientId` (used for `--oidc-client-id`) fields were already there in the integration.yaml and preview.yaml helm files. The `server` field I've added as part of this PR. Do I need to do something to ensure it works in production? (I cannot find the `issuer` or `clientId` fields in the production helmfile.yaml, so I am not quite sure where these values come from in production)

Reply:
I will make sure that this is added for production. You don't need to update anything else :)

Reply:
Lovely, thanks!